client/blocks/reader-post-card/gallery.jsx

```diff
@@ -44,10 +45,24 @@ const PostGallery = ( { post } ) => {
 		);
 	} );
 	return (
-		<ul className="reader-post-card__gallery">
-			{ listItems }
-		</ul>
-	);
+		<div className="reader-post-card__post" >
+			<ul className="reader-post-card__gallery">
+				{ listItems }
+			</ul>
+			<div className="reader-post-card__post-details">
+				<AutoDirection>
+					<h1 className="reader-post-card__title">
+						<a className="reader-post-card__title-link" href={ post.URL }>{ post.title }</a>
+					</h1>
+				</AutoDirection>
+				<AutoDirection>
+					<div className="reader-post-card__excerpt"
+						dangerouslySetInnerHTML={ { __html: post.better_excerpt || post.excerpt } } // eslint-disable-line react/no-danger
+					/>
+				</AutoDirection> )
```

Comment:
A rogue right bracket has slipped into the output here:

<img width="246" alt="screen shot 2017-01-12 at 14 05 28" src="https://cloud.githubusercontent.com/assets/17325/21878932/6c1ff4e8-d8d0-11e6-8800-297d9cfa8518.png">
<img width="391" alt="screen shot 2017-01-12 at 14 06 23" src="https://cloud.githubusercontent.com/assets/17325/21878933/6c204d80-d8d0-11e6-8e36-eae70e86cf36.png">